bundles/org.openhab.binding.kostalinverter/src/main/java/org/openhab/binding/kostalinverter/internal/secondgeneration/SecondGenerationHandler.java

```diff
@@ -92,14 +92,24 @@ public void handleCommand(ChannelUID channelUID, Command command) {
                 case SecondGenerationBindingConstants.CHANNEL_BATTERYUSAGECONSUMPTIONSET:
                     valueConfiguration = command.toString();
                     dxsEntriesConf = "33556249";
-                    preSetExecuteConfigurationChanges(httpClient, url, username, password, dxsEntriesConf,
-                            valueConfiguration);
+                    try {
+                        preSetExecuteConfigurationChanges(httpClient, url, username, password, dxsEntriesConf,
+                                valueConfiguration);
+                    } catch (Exception handleCommandException) {
+                        logger.debug("Handle command for {} on channel '{}: {}'", thing.getUID(), httpClient,
+                                handleCommandException.getMessage());
+                    }
                     break;
                 case SecondGenerationBindingConstants.CHANNEL_BATTERYUSAGESTRATEGYSET:
                     valueConfiguration = command.toString();
                     dxsEntriesConf = "83888896";
-                    preSetExecuteConfigurationChanges(httpClient, url, username, password, dxsEntriesConf,
-                            valueConfiguration);
+                    try {
+                        preSetExecuteConfigurationChanges(httpClient, url, username, password, dxsEntriesConf,
+                                valueConfiguration);
+                    } catch (Exception handleCommandException) {
```

Comment:
Please avoid catching a generic Exception but rather list more specific exceptions.
Same for all your added "catch"